docs/gaia/delegator-guide-cli.md

```diff
@@ -0,0 +1,388 @@
+
+::: danger
+**NEVER LOOSE YOUR MNEMONIC. WRITE IT DOWN ON A PIECE OF PAPER AND STORE IT SOMEWHERE SAFE. IF YOU LOOSE IT, THERE IS NO WAY TO RETRIEVE IT. IF SOMEONE GAINS ACCESS TO IT, THEY GAIN ACCESS TO ALL THE ASSOCIATED ACCOUNTS.**
+:::
+
+The address is a human-readble public information (e.g. `cosmos10snjt8dmpr5my0h76xj48ty80uzwhraqalu4eg`) that identifies your account. When someone wants to send you fund, they send it to your address. It is not possible to find the private key associated with a given address. 
+
+### Restoring an account from the fundraiser
+
+::: tip
+*NOTE: This section only concerns fundraiser participants*
+:::
+
+If you participated in the fundraiser, you should be in possession of a 12-words mnemonic. Newly generated mnemonic use 24 words, but 12-words mnemonic are also compatible with all the Cosmos tools. 
+
+#### On a ledger device
+
+At the core of a ledger device, there is a mnemonic used to generate accounts on multiple blockchains (including the Cosmos Hub). Usually, you will create a new mnemonic when you initialize your ledger device. However, it is possible to tell the ledger device to use a mnemonic provided by the user instead. Let us go ahead and see how you can input the mnemonic you obtained during the fundraiser as the seed of your ledger device. 
+
+::: warning
+*NOTE: To do this, **it is preferrable to use a brand new ledger device.**. Indeed, there can be only one mnemonic per ledger device. If however you want to use a ledger  that is already initalized with a seed, you can reset it by going in `Settings`>`Device`>`Reset All`. **Please note that this will wipe out the seed currently stored on the device. If you have not properly secured the associated mnemonic, you could loose your funds!!!***
+:::
+
+The following steps need to be performed on an un-initialized ledger device:
+
+1. Connect your ledger device to the computer via USB
+2. Press both buttons
+3. Do **NOT** choose the "Config as a new device" option. Instead, choose "Restore Configuration"
+4. Choose a PIN
+5. Choose the 12 words option
+6. Input each of the words you got during the fundraiser, in the correct order. 
+
+Your ledger is now correctly set up with your fundraiser mnemonic! Do not loose this mnemonic! If your ledger is compromised, you can always restore a new device again using the same mnemonic.
+
+Next, click [here](#using-a-ledger-device) to generate an account. 
+
+#### On a computer
+
+::: warning
+**NOTE: It is more secure to perform this action on an offline computer**
+::: 
+
+To restore an account using a fundraiser mnemonic and store the associated encrypted private key on a computer, use the following command:
```

Comment:
Make a note somewhere in this section that the key will be stored in a database held, by default at `~/.gaiacli/keys/`

Reply:
Maybe also make a note about terminal history (i.e. anywhere your mnemonic could potentially be stored in plain text.)